### PR TITLE
Tidy up for flow orgs

### PR DESCRIPTION
**Pull request by franklywatson**
Cleanup to remove orgs that have since been removed, or repos that are deleted. Also moved a bunch of individual repo listings into orgs

### CODE REVIEW

File: data/ecosystems/f/flow.toml [hunk 1]
```diff
@@ -69,27 +82,6 @@ github_organizations = [
 ]
 
 # Repositories
-[[repo]]
-url = "https://github.com/0xAlchemist/Athletaverse"
```
Comment (eherrerosj) [inline data/ecosystems/f/flow.toml:73]: why are you requesting to remove these ones?

Comment (franklywatson) [reply data/ecosystems/f/flow.toml:73]: You have linked one item, but spoke in plural. Was there more than one? 

For the one shown looks like i removed that in error. Let me fix that

Comment (eherrerosj) [reply data/ecosystems/f/flow.toml:73]: Yeah, I'm referring to all the repos that you removed.

In your PR, there's 400 flow repos that are gone when I compare them with the original file (~1900 vs ~1500). Am I missing something?

Comment (franklywatson) [reply data/ecosystems/f/flow.toml:73]: Yeah I moved a bunch of the individual items in the lower list into orgs where I know these are Flow repos. However, from looking at the PR there's something bad about the way I edited this which makes the change diff look really horrible and makes it hard to see what I am explaining. I will cancel this and make a cleaner PR so you can what was changed more directly

**Pull request closed without merging**